api/control/v1alpha1/types.go

```diff
@@ -260,11 +260,19 @@ type BindingSpec struct {
 type DownsyncObjectReferences struct {
```

Comment:
This type name is still misleading. Something like `DownsyncObjectReferencesWithStatusCollectors` would be more accurate.